kuma/static/js/experiment-static-examples-on-top.js

```diff
@@ -0,0 +1,14 @@
+(function() {
```

Comment:
We miss this all the time on bedrock, but if we're passing in `window.Mozilla` to the IIFE, it should be aliased here: `(function(Mozilla) {`.